### PR TITLE
Doc: Community Steering Committee policy: add topics triage workflow

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Doc: Community Steering Committee policy: add topics triage workflow

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/community/steering/community_steering_committee.rst

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @felixfontein @cybette @acozine @samccann @briantist @mariolenz

**Review by briantist — APPROVED** [review]

File: docs/docsite/rst/community/steering/community_steering_committee.rst [hunk 1]
```diff
@@ -106,6 +106,26 @@ In case of equal numbers of votes for and against a topic, the chairperson's vot
 
 For votes with more than two options, one choice must have at least half of the votes. If two choices happen to both have half of the votes, the chairperson's vote will break the tie. If no choice has at least half of the votes, the vote choices have to be adjusted so that a majority can be found for a choice in a new vote.
 
+Community topics triage
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The Committee conducts a triage of `community topics <https://github.com/ansible-community/community-topics/issues>`_ periodically (every three-six months).
```
Comment (samccann) [inline docs/docsite/rst/community/steering/community_steering_committee.rst]:
```suggestion
The Committee conducts a triage of `community topics <https://github.com/ansible-community/community-topics/issues>`_ periodically (every three to six months).
```

File: docs/docsite/rst/community/steering/community_steering_committee.rst [hunk 2]
```diff
@@ -106,6 +106,26 @@ In case of equal numbers of votes for and against a topic, the chairperson's vot
...
+
+The triage goals are:
+
+* Sparking interest to forgotten topics.
```
Comment (samccann) [inline docs/docsite/rst/community/steering/community_steering_committee.rst]:
```suggestion
* Sparking interest for forgotten topics.
```

**Comment by Andersson007** [comment]
@briantist @samccann thanks for reviewing!
@samccann feel free to merge, thanks

**Review by samccann — APPROVED** [review]

**Pull request merged**